main

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env ruby
+
+require './teacher'
+require './student'
+require './book'
+require './rental'
```

Comment:
[OPTIONAL] I think It's a good practice for the Ruby programmers to import their files with `require_relative` when they are not imported from gems or installed packages. [You can read more about good practices for Ruby here.](https://rubystyle.guide/)